src/app/features/auth/pages/forgot-password/forgot-password.component.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, OnDestroy, computed, effect, inject, signal } from '@angular/core';
+import { FormControl, FormGroup, FormsModule, NonNullableFormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+
+import { NzFormModule } from 'ng-zorro-antd/form';
+import { NzInputModule } from 'ng-zorro-antd/input';
+import { NzButtonModule } from 'ng-zorro-antd/button';
+import { NzGridModule } from 'ng-zorro-antd/grid';
+import { NzTypographyModule } from 'ng-zorro-antd/typography';
+import { NzSpaceModule } from 'ng-zorro-antd/space';
+import { Router, RouterModule } from '@angular/router';
+import { AuthService } from '../../../../shared/services/auth.service';
+import { EMPTY, Subject, catchError, finalize, takeUntil } from 'rxjs';
+import { NzMessageModule, NzMessageService } from 'ng-zorro-antd/message';
+
+@Component({
+  selector: 'app-forgot-password',
+  standalone: true,
+  imports: [NzFormModule, NzInputModule, NzButtonModule, FormsModule, ReactiveFormsModule, NzGridModule, NzTypographyModule, NzSpaceModule, RouterModule, NzMessageModule],
+  templateUrl: './forgot-password.component.html',
+  styleUrl: './forgot-password.component.sass'
+})
+export class ForgotPasswordComponent implements OnDestroy {
+  private router = inject(Router);
+  private authService = inject(AuthService)
+  private fb = inject(NonNullableFormBuilder)
+  private toast = inject(NzMessageService)
+
+  private loadingState = signal<boolean>(false)
+
+  onDestroy$: Subject<void> = new Subject();
+
+  isLoading = computed(() => this.loadingState())
+
+  validateForm: FormGroup<{
+    email: FormControl<string>;
```

Comment:
Don't we need control for password?
If this form is for logging in let's rename it for `loginForm` or smth else, so that we could understand what this form is responsible for